### ISSUE TITLE
filename case issue with lang/R/tests/testthat/TestInterface.R

### ISSUE THREAD

**Issue by schristley** [airr-community/airr-standards#113]
I saw this the other day when trying to clean up the airr-tools branch, so I pulled a fresh clone and there seems to be an issue with case for a file.

```
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   lang/R/tests/testthat/TestInterface.R

no changes added to commit (use "git add" and/or "git commit -a")
```

The file `TestInterface.R` does not exist, but `testInterface.R` does exist. I'm on a mac which is case-insensitive for filenames, so these are technically the same files.

If I made a guess, the file was `git add` as `TestInterface.R` but file created on disk is `testInterface.R` (or vice-versa) and it's confusing git. Not sure how to fix though, doing a `git add` doesn't do anything, nor does `git checkout --`

```
$ git add lang/R/tests/testthat/testInterface.R
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   lang/R/tests/testthat/TestInterface.R

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by ssnn-airr** [comment]
testThat didn't recognize the capital case file
I think I used git mv

From: Scott Christley
Sent: Friday, April 13, 6:00 PM
Subject: [airr-community/airr-standards] filename case issue with lang/R/tests/testthat/TestInterface.R (#113)
To: airr-community/airr-standards
Cc: Subscribed


I saw this the other day when trying to clean up the airr-tools branch, so I pulled a fresh clone and there seems to be an issue with case for a file.
$ git status On branch master Your branch is up-to-date with 'origin/master'. Changes not staged for commit: (use "git add <file>..." to update what will be committed) (use "git checkout -- <file>..." to discard changes in working directory) modified: lang/R/tests/testthat/TestInterface.R no changes added to commit (use "git add" and/or "git commit -a")
The file TestInterface.R does not exist, but testInterface.R does exist. I'm on a mac which is case-insensitive for filenames, so these are technically the same files.
If I made a guess, the file was git add as TestInterface.R but file created on disk is testInterface.R (or vice-versa) and it's confusing git. Not sure how to fix though, doing a git add doesn't do anything, nor does git checkout --
$ git add lang/R/tests/testthat/testInterface.R $ git status On branch master Your branch is up-to-date with 'origin/master'. Changes not staged for commit: (use "git add <file>..." to update what will be committed) (use "git checkout -- <file>..." to discard changes in working directory) modified: lang/R/tests/testthat/TestInterface.R no changes added to commit (use "git add" and/or "git commit -a")
